### DISCUSSION TITLE
Chat synchronisation makes keeping track of messages difficult

### DISCUSSION THREAD

**Discussion by Pennek** [ppy/osu#13166] [Ideas]
Follow-up to #10540

Because `osu!web` and `osu!lazer` messages are synchronised, community members who have to interact with a large amount of people have to choose between cluttering their [lazer chat](https://youtu.be/GU6-UCGT5XQ) to keep track of their messages and message-history, or closing in-game conversations to hopefully remember the names of the people they're expecting answers from.

I was hoping we could figure out some way to keep web-side chat history even after closing in-game tabs

**Comment by peppy** [comment]
They are intentionally in sync. If anything the UX for tabs will improve (and match the web implementation).
